specification/storsimple8000series/resource-manager/readme.md

```diff
@@ -37,6 +37,10 @@ These settings apply only when `--tag=package-2017-06` is specified on the comma
 ``` yaml $(tag) == 'package-2017-06'
 input-file:
 - Microsoft.StorSimple/stable/2017-06-01/storsimple.json
+directive:

```

Comment:
Has this been approved by @salameer or someone from the review committee?

Reply:
Hey Deepak
I looped you in to the email thread where suppression was granted. :)